lds/guest.lds

```diff
@@ -41,7 +41,7 @@ SECTIONS
     } >ram AT>ram :bss
 
     PROVIDE(_stack_start = _bss_end);
-    PROVIDE(_stack_end = _stack_start + 0x6000);
+    PROVIDE(_stack_end = _stack_start + 0x30000);
```

Comment:
A little concerning that such a simple program blew past 24kB of stack. I haven't been following too closely, but is this because certificate parsing is using large stack-allocated arrays? If we anticipate Tellus being similarly stack-hungry, we need to bump up how much we're allocating for secondary CPU stacks.

Reply:
I dont anticipate Tellus being that stack hungry. 

The certificate parsing is all stack-allocated array based, and some of the structures there are quite nested, which explains why guestvm needs that stack bump.

Note that we're calling into the attestation crate from `guestvm` for debug/verification purposes, as we're deserializing the generated certificate to print some of its attributes. This all comes from the `Certificate::from_der` call.

Reply:
> On a related note, are we planning on moving some of this certificate processing to the TSM? If so, do we need to be concerned about the stack space requirements there as well?

We should be concerned, but fwiw the TSM is already doing the certificate processing (CSR parsing and certificate creation).

Reply:
The secondary CPU stacks are currently just under 16kB. If we think that's insufficient for certificate handling, let's bump up `PER_CPU_PAGES` in smp.rs.

Reply:
> The secondary CPU stacks are currently just under 16kB. If we think that's insufficient for certificate handling, let's bump up `PER_CPU_PAGES` in smp.rs.

I bumped them to 8 pages, it should be fine for now.